### ISSUE TITLE
New field never loads in 2.0.3

### ISSUE THREAD

**Issue by tidy** [engram-design/FieldManager#54]
### Description
Since upgrading to v2.0.3 on Craft 3, when I click the 'new field' button in the top right, the pop-up content never loads (the loading icon just spins forever)

### Additional info

- Plugin version: 2.0.3
- Craft version: 3.0.26.1

**Comment by matthisamoto** [comment]
I am also having this issue. Errors posted below:

console errors:
```
POST [url]/index.php?p=admin/actions/field-manager/base/get-field-modal-body 500 (Internal Server Error)

field-manager.js:353 Uncaught TypeError: Cannot read property 'success' of null
    at s.constructor.<anonymous> (field-manager.js:353)
```
server error:
```
{"error":"Argument 1 passed to craft\\services\\Fields::getFieldById() must be of the type integer, string given, called in [path]/vendor/verbb/field-manager/src/controllers/BaseController.php on line 80"}
```

**Comment by engram-design** [comment]
Fixed in [2.0.4](https://github.com/verbb/field-manager/releases/tag/2.0.4)
